src/components/SolidLoginForm.jsx

```diff
@@ -1,38 +1,63 @@
 import { getDefaultSession, fetch } from "@inrupt/solid-client-authn-browser";
-import { getLiteral, getProfileAll, getThing } from "@inrupt/solid-client";
+import {
+  getLiteral,
+  getProfileAll,
+  getThing,
+} from "@inrupt/solid-client";
 import { FOAF } from "@inrupt/vocab-common-rdf";
 import { useState } from "react";
+import { QueryEngine } from "@comunica/query-sparql";
 
 function SolidLoginForm(props) {
   const session = getDefaultSession();
   let webId = session.info.webId;
   const [name, setName] = useState(undefined);
+  const [errorMessage, setErrorMessage] = useState(undefined);
+
   if (webId) {
-    getProfileAll(webId, { fetch: fetch }).then((dataSet) => {
-      let profile = dataSet.webIdProfile;
-      const webIdThing = getThing(profile, webId);
-      let literalName = getLiteral(webIdThing, FOAF.name);
-      if (literalName) {
-        setName(literalName.value);
-      } else {
-        setName(webId);
-      }
-    }).catch(_ => setName(webId));
+    getProfileAll(webId, { fetch: fetch })
+      .then((dataSet) => {
+        let profile = dataSet.webIdProfile;
+        const webIdThing = getThing(profile, webId);
+        let literalName = getLiteral(webIdThing, FOAF.name);
+        if (literalName) {
+          setName(literalName.value);
+        } else {
+          setName(webId);
+        }
+      })
+      .catch((_) => setName(webId));
   }
 
   /**
    * Handling what should happen when the user trying to log in.
```

Comment:
Fix grammar.

Reply:
Adjusted this in [commit aa49f71](https://github.com/SolidLabResearch/generic-data-viewer/pull/52/commits/aa49f7162c561eb7318bd6f385f79d07b3942001)